rust/worker/src/execution/operators/brute_force_knn.rs

```diff
@@ -169,9 +166,7 @@ impl Operator<BruteForceKnnOperatorInput, BruteForceKnnOperatorOutput> for Brute
             // Empty allowed list is passed when where filtering is absent.
             // TODO: This should not need to use merged_user_id, which clones the id.
```

Comment:
i think we might be able to use merged_user_id_ref

Reply:
Can't convert &str to &String